ChangeLog.md

```diff
@@ -10,6 +10,12 @@ Behaviour changes:
 
 Other enhancements:
 
+* A new sub command `ls` has been introduced to stack to view
+  local and remote snapshots present in the system. Use `stack ls
+  snapshots --help` to get more details about it.
+* `pageText` function introduced in `System.Process.PagerEditor`
```

Comment:
Implementation details shouldn't go in the changelog, since stack isn't typically used as a library.  All of stack's APIs should be considered to be "internal", in other words they are subject to change even between minor versions.

Reply:
@mgsloan Fixed in the latest push.